### DISCUSSION TITLE
Sync to MIDI clock

### DISCUSSION THREAD

**Discussion by cliffspencer** [publicsamples/Oi-Grandad#22] [Q&A] [answered]
I'm using Oi Grandad in Reaper. I'd like to sync the modulation cycles to the MIDI clock, how is this done? Thanks!

**Comment by RickTaylor79** [comment, marked as answer]
The modulators are already synced to the daws clock. This is the default (and only) setting. 

If you want to sync it to the transport, then you need to set the modulator trig value to "note" and then record some midi into your daw. This will trigger the modulator each time the note is played.

**Reply by cliffspencer** [reply]
Got it. Thanks!
